code/modules/power/power.dm

```diff
@@ -355,7 +355,7 @@
 	if(H)
 		if(H.species.siemens_coefficient == 0)
 			return
-		if(isvaurca(H)) //natural vaurca insulation
+		if(isvaurca(H) || H.is_diona()) //natural vaurca and dionae insulation
```

Comment:
Nooooo. If you're doing this, make this a species proc such as:

`/datum/species/proc/is_naturally_insulated()` which returns FALSE by default and returns TRUE on vaurca/diona.

Reply:
Changed, works in testing